apps/server/src/modules/task/repo/task-scope.ts

```diff
@@ -1,5 +1,5 @@
 import { EntityId, Task } from '@shared/domain';
-import { Scope } from '@shared/repo';
+import { Scope } from '@shared/repo/scope'; // TODO move this file to shared
```

Comment:
What do you mean by this todo? The file is already in the shared directory?

Reply:
deep imports should be avoided, while the task repo will be moved in a next chunk as part of the ticket